src/components/CountrySelector/index.tsx

```diff
@@ -68,7 +69,7 @@ export default function CountrySelector<
       as="div"
       className={`relative items-center grid grid-cols-[auto_auto_1fr] w-full field-container ${
         props.classes?.container || ""
-      }`}
+      } ${getErrorClasses(errors, `${props.fieldName}.name`)}`}
```

Comment:
@abbasali1994
Another way to attach these error classes would be to add aria attribute

```ts
const isError = !!errors[name /** usually provided*/]?.message
<Combobox aria-invalid={isError}/>
```
and in `index.css` could put classes in `field-container` and `field-input` utilities
```
@apply ... aria-invalid:border-red dark:aria-invalid:border-red

```
this way no need to generalize gettng error info as the nesting depends on every component as shown in this function

```
export const resolvePath = (object: any, path: string) =>
  path.split(".").reduce((o, p) => (o ? o[p] : {}), object);
```

Reply:
Done